nvflare/apis/storage.py

```diff
@@ -217,4 +217,9 @@ def tag_object(self, uri: str, tag: str, data=None):
 
     @staticmethod
     def is_valid_component(component_name):
-        return component_name in [DATA, META, WORKSPACE]
+        valid_components = {DATA, META, WORKSPACE, "ERRORLOG", "LOG"}
```

Comment:
For "LOG" and "ERRORLOG" do we want to define some constant variable for these strings, and could there be more log types?

Reply:
Yes, this is just to put something for now, and we can certainly add more, so I will create a variable for this.